README.md

```diff
@@ -1,6 +1,6 @@
 # OpenJFX Docs
 
-OpenJFX docs contains documentation for "Getting Started with JavaFX 11".
+OpenJFX docs contains documentation for "Getting Started with JavaFX".
```

Comment:
This removes the need for updating the version every 6 months, but it doesn't clarify what JavaFX version is supported. At least it should be `11+` ?